src/wallet.ts

```diff
@@ -1,32 +1,47 @@
-export default {
-  "0x435933c8064b4ae76be665428e0307ef2ccfbd68": {
+type Account = {
+  privateKey: string;
+  address: string;
+  comment: string;
+  balance?: string;
+};
+
+const accounts: { [key: string]: Account } = {
+  "account0": {
+    privateKey: "",
+    address: "0x7567d83b7b8d80addcb281a71d54fc7b3364ffed",
+    balance: "0x14ADF4B7320334B9000000",
+    comment: "Account with no energy",
+  },
+  "account1": {
     privateKey:
       "7b067f53d350f1cf20ec13df416b7b73e88a1dc7331bc904b92108b1e76a08b1",
     address: "0x435933c8064b4ae76be665428e0307ef2ccfbd68",
     comment: "TODO",
   },
-  "0x0f872421dc479f3c11edd89512731814d0598db5": {
+  "account2": {
     privateKey:
       "f4a1a17039216f535d42ec23732c79943ffb45a089fbb78a14daad0dae93e991",
     address: "0x0f872421dc479f3c11edd89512731814d0598db5",
     comment: "TODO",
   },
-  "0xf370940abdbd2583bc80bfc19d19bc216c88ccf0": {
+  "account3": {
     privateKey:
       "35b5cc144faca7d7f220fca7ad3420090861d5231d80eb23e1013426847371c4",
     address: "0xf370940abdbd2583bc80bfc19d19bc216c88ccf0",
     comment: "TODO",
   },
-  "0x99602e4bbc0503b8ff4432bb1857f916c3653b85": {
+  "account4": {
     privateKey:
       "10c851d8d6c6ed9e6f625742063f292f4cf57c2dbeea8099fa3aca53ef90aef1",
     address: "0x99602e4bbc0503b8ff4432bb1857f916c3653b85",
     comment: "TODO",
   },
-  "0x61e7d0c2b25706be3485980f39a3a994a8207acf": {
+  "account5": {
     privateKey:
       "2dd2c5b5d65913214783a6bd5679d8c6ef29ca9f2e2eae98b4add061d0b85ea0",
     address: "0x61e7d0c2b25706be3485980f39a3a994a8207acf",
     comment: "TODO",
   },
 };
+
+export const wallet = (name: string) => { return accounts[name] }
```

Comment:
This will be much cleaner if you do:

```
type WalletKey = keyof typeof accounts

export const wallet = (name: WalletKey) => { return accounts[name] }

```

This provides typed intellisense

Reply:
<img width="1044" alt="image" src="https://github.com/vechain/thor-e2e-tests/assets/107671032/38001213-be3b-438a-840b-c390010a5a99">

Reply:
done, thanks!

Reply:
Gonna push one minor as intellisense isn't working

Reply:
This commit keeps the typing of the wallet record, by enforcing a return type on `wallet(..)` function

https://github.com/vechain/thor-e2e-tests/pull/6/commits/d426e8e59df34061f31db75f8959e4115a7b1bff